ts/test/util.test.ts

```diff
@@ -18,6 +18,19 @@ test("dhtSync - Create multiple entries, read the last, 2 conductors", async ()
     { appBundleSource },
   ]);
 
+  // Alice and Bob init their cells
+  // This is a workaround for https://github.com/holochain/holochain/issues/5363
```

Comment:
Should this be a TODO since the problem is being resolved and this code can likely be removed again relatively soon?

Reply:
Actually I was confused here -- this is not a workaround for the linked issue, it just causes the InitZomesComplete ops to be created. Updated the comment to clarify: https://github.com/holochain/tryorama/pull/303/commits/ce43233d1fafc8ccf85b66360ac86fde6f51afa3

I did add TODO to the other workaround comments related to conductor launch staggering here: https://github.com/holochain/tryorama/pull/303/commits/21be33b6b61f60580961ea5fad3cf0558e98eee4